Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.yml

```diff
@@ -423,8 +423,9 @@ script:
   longRunningPort: false
   runonce: false
   script: '-'
-  subtype: python3
+  subtype: python2
```

Comment:
Why did we change this? what was the error you got? This is rather new integration, I don't think it should be py2

Reply:
The error was `AttributeError: 'dict' object has no attribute 'iteritems'`
https://app.circleci.com/pipelines/github/demisto/content/97450/workflows/f8569a69-7fb7-4c2a-8cb6-8f02de4f66c4/jobs/391166